tests/sota_checkpoints_eval.json

```diff
@@ -10,13 +10,24 @@
             "resnet50_imagenet_int8": {
                 "config": "examples/classification/configs/quantization/resnet50_imagenet_int8.json",
                 "reference": "resnet50_imagenet",
-                "target": 75.97,
+                "target": 76.4,
```

Comment:
Per today's discussions we will have @AlexKoff88 and @alexsu52 reviewing changes to the reference metrics files in any PR that does them, including this one.

Reply:
okay